diff --git a/tensorflow/workspace2.bzl b/tensorflow/workspace2.bzl
index 059a5921..befe43c6 100644
--- a/tensorflow/workspace2.bzl
+++ b/tensorflow/workspace2.bzl
@@ -127,11 +127,11 @@ def _tf_repositories():
     # and update the sha256 with the result.
     tf_http_archive(
         name = "XNNPACK",
-        sha256 = "5482fb0fcdc1df8b4842f8edf944443ea67ffe712a5cd846f0af484abe4f9a79",
-        strip_prefix = "XNNPACK-8f15372eb67ffab0d54cfe3752acaf8f8415af17",
+        sha256 = "38f9fd8c91acf3bdeff695117fa184e4d510d12d8fb320b4430e21c0f9e686b5",
+        strip_prefix = "XNNPACK-0d979d5ee62c750d122235a11e4568ef04a50243",
         urls = [
-            "https://storage.googleapis.com/mirror.tensorflow.org/github.com/google/XNNPACK/archive/8f15372eb67ffab0d54cfe3752acaf8f8415af17.zip",
-            "https://github.com/google/XNNPACK/archive/8f15372eb67ffab0d54cfe3752acaf8f8415af17.zip",
+            "https://storage.googleapis.com/mirror.tensorflow.org/github.com/google/XNNPACK/archive/0d979d5ee62c750d122235a11e4568ef04a50243.zip",
+            "https://github.com/google/XNNPACK/archive/0d979d5ee62c750d122235a11e4568ef04a50243.zip",
         ],
     )
 
